src/bosh_openstack_cpi/spec/unit/create_vm_spec.rb

```diff
@@ -42,14 +42,25 @@ def openstack_params(network_spec = { "network_a" => dynamic_network_spec}, boot
     }
 
     if boot_from_volume
-      params[:block_device_mapping_v2] = [{
-        :uuid => "sc-id",
-        :source_type => "image",
-        :dest_type => "volume",
-        :volume_size => 2048,
-        :boot_index => "0",
-        :delete_on_termination => "1",
-        :device_name => "/dev/vda" }]
+      if using_root_disk
+        params[:block_device_mapping_v2] = [{
+          :uuid => "sc-id",
+          :source_type => "image",
+          :dest_type => "volume",
+          :volume_size => 10240,
```

Comment:
can we make volume_size a parameter of the `openstack_params` method instead of `using_root_disk`? This would get rid of the magic numbers `10240` and `2048` here and also lose the nasty second `if`